.make.defaults

```diff
@@ -578,12 +578,12 @@ MINIO_ADMIN_PWD= localminiosecretkey
 	@# Help: Update pyproject.toml to depend on lib versions defined in .make.versions 
 	@if [ -e pyproject.toml ]; then					\
 	    cat pyproject.toml | sed 									\
-		-e 's/"data-prep-toolkit-ray\(..\).*",/"data-prep-toolkit-ray\1$(DPK_LIB_VERSION)",/' 	\
-		-e 's/"data-prep-toolkit-spark\(..\).*",/"data-prep-toolkit-spark\1$(DPK_LIB_VERSION)",/' 	\
-		-e 's/"data-prep-toolkit-kfp\([=><][=><]\).*",/"data-prep-toolkit-kfp\1$(DPK_LIB_KFP_VERSION)",/' 	\
-		-e 's/"data-prep-toolkit\([=><][=><]\).*",/"data-prep-toolkit\1$(DPK_LIB_VERSION)",/' 		\
-		-e 's/"ray\[default\]\([=><][=><]\).*",/"ray\[default\]\1$(RAY)",/'				\
-		-e 's/"data-prep-toolkit-kfp-shared\(..\).*",/"data-prep-toolkit-kfp-shared\1$(DPK_LIB_KFP_VERSION)",/'     \
+		-e 's/"data-prep-toolkit-ray\([=><][=><]\).*"/"data-prep-toolkit-ray\1$(DPK_LIB_VERSION)"/' 	\
```

Comment:
if you want to narrow possible options, I'd define it as  `[=><~][=].

Reply:
Thanks and done.